blog

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="stylesheet" type="text/css" href="styles.css">  
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.1/css/all.css" integrity="sha384-vp86vTRFVJgpjF9jiIGPEEqYqlDwgyBgEF109VFjmqGmIY/Y4HV4d3Gp2irVfcrp" crossorigin="anonymous">
+  <title></title>
+</head>
+
+<body>
+
+<header>
+<div class="header-holder">
+<section class="left-side-header">
+<img src="images/adoptadog-logo.svg">
+<p>Seattle's premiere dog adoption agency</p>
+</section>
+<section class="right-side-header">
+  <p><i class="fas fa-shopping-cart"></i> $1234.56  </p>
+  <button> Checkout</button>
+    
+    </section>
+</div>    
+</header>
+<nav>
+<ul class="header-nav-list-holder">
+    <li class="header-nav-list-item"><a href="#home">Home</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="#dogs">Dogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    <li class="header-nav-list-item"><a href="#blogs">Blogs</a></li>
+    <li class="header-nav-list-item">/</li>
+    </ul>
+    
+    </nav>    
+<!--<div id="wrapper">-->
+<h2 class="dogs-list-header">Here are all our Dogs!</h2>
+    <div class="grid-item-container"> 
+
+<section class="dog-holder">
```

Comment:
You'll want to be more consistent with your identation. It'll make your code much easier to use. Most code editors have an auto-ident feature that should align your code for you